src/schemas/index.ts

```diff
@@ -1,9 +1,23 @@
+import { McpbManifestSchema as ManifestSchemaV0_1 } from "./0.1.js";
+import { McpbManifestSchema as ManifestSchemaV0_2 } from "./0.2.js";
+import { McpbManifestSchema as ManifestSchemaV0_3 } from "./0.3.js";
+
 export * as v0_1 from "./0.1.js";
 export * as v0_2 from "./0.2.js";
 export * as v0_3 from "./0.3.js";
-export * as any from "./any.js";
-export * as latest from "./latest.js";
+export * as vAny from "./any.js";
+export * as vLatest from "./latest.js";
```

Comment:
I think we should remove any concept of "latest" from the exported types. We're going to accidentally depend on latest when basically every consumer should be depending on `vAny` / the union type always

Reply:
Like, in this case isn't `vLatest` actually `vNext`?

Reply:
exporting N schemas is a fine choice too, I just need to be able to grab the schema for a given version